package.json

```diff
@@ -5,8 +5,12 @@
   "private": true,
   "scripts": {
     "dev": "NODE_ENV=development DEBUG=metalsmith-timer,metalsmith-algolia,metalsmith-layouts,metalsmith-assets node index.js",
-    "build": "NODE_ENV=production DEBUG=metalsmith-timer,metalsmith-algolia node index.js",
-    "webpack": "webpack --progress --colors"
+    "build": "DEBUG=metalsmith-timer,metalsmith-algolia node index.js",
+    "webpack": "webpack --progress --colors",
+    "build:production": "NODE_ENV=production npm run build # no drafts, no beta, no archive",
+    "build:preview": "NODE_ENV=preview npm run build # includes drafts, beta and archive",
```

Comment:
doesn't look like you actually use the `preview` value anywhere in your code, so could be removed.